librecad/src/actions/rs_actionpolylinesegment.cpp

```diff
@@ -54,9 +54,9 @@ void RS_ActionPolylineSegment::init(int status) {
 	if (container->countSelected(true, entityType)) {
 		//find a selected entity
 		//TODO, find a better starting point
-		for (RS_Entity* e = container->firstEntity(RS2::ResolveAllButInserts);
+		for (RS_Entity* e = container->firstEntity(RS2::ResolveNone);
```

Comment:
Do you agree that it needs to be ResolveNone though? I did a quick look at the types of entities that exist, and I don't think any of those needs to be descended into - certainly not blocks, because they're supposed to be separate entities and not to be changed by any operations on the main drawing. Maybe I missed something there, though.